task/06-conditions-n-loops-tasks.js

```diff
@@ -431,10 +517,29 @@ function getMatrixProduct(m1, m2) {
  *
  */
 function evaluateTicTacToePosition(position) {
-    throw new Error('Not implemented');
+   let tmpArr = position;
+   let wins  =[[0,1,2],[3,4,5],[6,7,8],[0,4,8],[2,4,6],[0,3,6],[1,4,7],[2,5,8]];
+    
+   for (let i = 0; i < 3; i++){
+    for (let j = 0; j < 3; j++){
+      if( tmpArr[i][j] != 'X' && tmpArr[i][j] != '0'){
```

Comment:
"operator !== is always preferable
sometimes ''!=' may cause an error
(check at console :
""0"" != 0
and ""0"" !== 0)"